static/app/views/starfish/modules/databaseModule/panel/index.tsx

```diff
@@ -394,7 +394,7 @@ const FlexRowItem = styled('div')`
   flex: 1;
 `;
 
-const FormattedCode = styled('div')`
+export const FormattedCode = styled('div')`
```

Comment:
We should probs move this outside of the db module code now that it's being used outside of it.

Reply:
Done!